content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -17,18 +19,18 @@ Poseidon is the [Firmament scheduler](https://github.com/Huawei-PaaS/firmament)
 
 ## Introduction	
 
-Poseidon is the [Firmament scheduler](https://github.com/Huawei-PaaS/firmament) integration for Kubernetes. Poseidon/Firmament scheduler augments the current Kubernetes scheduling capabilities. It incorporates new novel flow network graph based scheduling capabilities alongside the default Kubernetes Scheduler.  
+Poseidon is the integration glue for the  [Firmament scheduler](https://github.com/Huawei-PaaS/firmament) with Kubernetes. Poseidon-Firmament scheduler augments the current Kubernetes scheduling capabilities. It incorporates novel flow network graph based scheduling capabilities alongside the default Kubernetes Scheduler. Firmament scheduler models workloads and clusters as flow networks and runs min-cost flow optimizations over these networks to make scheduling decisions.
 
-It models the scheduling problem as a constraint-based optimization over a flow network graph. This is achieved by reducing scheduling to a min-cost max-flow optimization problem. Due to the inherent rescheduling capabilities, the Poseidon/Firmament scheduler constantly keeps refining the workloads placements dynamically.  
+It models the scheduling problem as a constraint-based optimization over a flow network graph. This is achieved by reducing scheduling to a min-cost max-flow optimization problem. Due to the inherent rescheduling capabilities, the Poseidon-Firmament scheduler constantly keeps refining the workloads placements dynamically.  
 
```

Comment:
Suggestion:
Remove, `Due to the inherent rescheduling capabilities`
Instead,
The Poseidon-Firmament scheduler dynamically refines the workload placements.

Reply:
This is done.